### DISCUSSION TITLE
what do I do after I clone the repo?

### DISCUSSION THREAD

**Discussion by nanoxmaster** [kenjyoung/MinAtar#24] [Q&A]
Sorry, I'm a newbie, I cloned the repo with git, after that I tried to put the command "pip install ." and an error appeared. I have to change the dot for something else?

**Comment by kenjyoung** [comment]
It's hard to say without more information what might be going wrong. If you post the error I might be able to help. If you navigate into the repo directory and do "pip install ." that should work if your environment is set up appropriately. Alternatively, you can do "pip install minatar" without cloning anything since the repo is now available on PyPi. Best of luck! 😀

**Reply by nanoxmaster** [reply]
thanks, I was able to resolve my error and install the package
